zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -339,6 +347,36 @@ def query_volume(self, req):
 
         return jsonobject.dumps(rsp)
 
+    @replyerror
+    def batch_query_volume(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.8.2)

`http` may be undefined, or defined from star imports

(F405)